src/SFML/Graphics/RenderTexture.cpp

```diff
@@ -183,6 +198,7 @@ Vector2u RenderTexture::getSize() const
 ////////////////////////////////////////////////////////////
 bool RenderTexture::isSrgb() const
 {
+    assert(m_impl && "Must call RenderTexture::create first");
```

Comment:
Should this remain an assert? If so, the message must be adjusted as `create` doesn't exist anymore.